src/components/PopupWithForm.js

```diff
@@ -10,7 +10,7 @@ export default class PopupWithForm extends Popup {
 
   close() {
     super.close();
```

Comment:
You shouldn’t create the **same methods** of the parent class if you don’t change them inside and only call them with `super`. The parent methods are available in any inheritor of the class.